browser/net/brave_reduce_language_network_delegate_helper.cc

```diff
@@ -0,0 +1,109 @@
+/* Copyright (c) 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include <string>
+#include <vector>
+
+#include "brave/browser/net/brave_reduce_language_network_delegate_helper.h"
+
+#include "base/strings/string_split.h"
+#include "brave/browser/brave_browser_process.h"
+#include "brave/components/brave_shields/browser/brave_farbling_service.h"
+#include "brave/components/brave_shields/browser/brave_shields_util.h"
+#include "brave/components/brave_shields/common/features.h"
```

Comment:
not used?

Reply:
Some of these are used in `FarbleAcceptLanguageHeader`. I removed the unused ones.